<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QWidget" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1399</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	font-size: 14px;
	padding: 5px;
	background-color: white;
	border: 1px solid black;
}
#fr_sinhvien, #fr_lophoc, #fr_thongtin{
	border: 2px solid grey;
}
QComboBox{
	font-size:14px;
	border: 1px solid black;
	padding: 5px
}
QComboBox::drop-down{
	border: none;
}
QComboBox::down-arrow{
	image: url(:/assets/assets/icons/droparrow.ico);
	width: 18px;
	background-color: white;
	height: 20px;
	margin-right: 15px
}
QComboBox:on
QComboBox QListView
QComboBox QListView::item
QComboBox QListView::item:hover
QComboBox QListView::item:selected</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_main">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>QUẢN LÝ THÔNG TIN SINH VIÊN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1400</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="fr_thongtin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>561</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{
	border: 1px solid black
}
QPushButton{
	padding: 15px 5px;
	background-color: #D5EDF9;
}
#gridLayout{
	border: 1px solid black;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="grbx_canhan">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>521</width>
       <height>591</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel, QPushButton{
	font-size: 14px;
}</string>
     </property>
     <property name="title">
      <string>Thông tin cá nhân</string>
     </property>
     <widget class="QLabel" name="lb_hoten">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mã SV:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_namsinh">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>90</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Năm sinh:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_cccd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>57</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CCCD:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_gioitinh">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Giới tính:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_email">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>56</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_sdt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>44</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SĐT:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_mssv">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_namsinh">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Nhập năm sinh</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_cccd">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Nhập căn cước công dân</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_email">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Nhập email</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_sdt">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Nhập số điện thoại</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_diachi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Địa chỉ:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_diachi">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>370</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Nhập địa chỉ</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_lophoc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lớp</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_hoten">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Nhập họ tên sinh viên</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_hoten_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Họ tên:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbx_gioitinh">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>221</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Nam</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nữ</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbx_lophoc">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QFrame" name="gridBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>491</width>
        <height>118</height>
       </rect>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>7</y>
         <width>481</width>
         <height>54</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>1</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_them">
          <property name="text">
           <string>Thêm mới</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_xoa">
          <property name="text">
           <string>Xóa</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_sua">
          <property name="text">
           <string>Sửa</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_reset">
          <property name="text">
           <string>Làm mới</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>481</width>
         <height>54</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_luu_2">
          <property name="text">
           <string>Lấy ảnh</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_traindata">
          <property name="text">
           <string>Training Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="lb_thongtin">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>10</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>THÔNG TIN SINH VIÊN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_lophoc">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>801</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	padding: 15px 5px;
	font-size:14px;
	background-color: #D5EDF9;
}
#lb_tenlop, #lb_malop{
	font-size:14px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="txt_tklop">
     <property name="geometry">
      <rect>
       <x>534</x>
       <y>60</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Tìm kiếm</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbx_tklophoc">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>74</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Mã lớp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tên lớp</string>
      </property>
     </item>
    </widget>
    <widget class="QTableWidget" name="tbClass">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>341</width>
       <height>201</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>99</number>
     </attribute>
     <column>
      <property name="text">
       <string>Mã lớp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tên lớp</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="lb_thongtin_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>771</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size: 20px;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>DANH SÁCH LỚP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_malop">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mã lớp:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tenlop">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tên lớp:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txt_malop">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>Nhập mã lớp</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txt_tenlop">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Nhập tên lớp</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="horizontalSpacing">
       <number>1</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_themlop">
        <property name="text">
         <string>Thêm mới</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_xoalop">
        <property name="text">
         <string>Xóa</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_sualop">
        <property name="text">
         <string>Sửa</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_resetlop">
        <property name="text">
         <string>Làm mới</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_sinhvien">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>60</y>
      <width>801</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tbStudents">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>781</width>
       <height>211</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Mã SV</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Họ tên</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mã lớp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CMND</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Giới tính</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ngày sinh</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SDT</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="lb_thongtin_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>781</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>DANH SÁCH SINH VIÊN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cbx_sortsv">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>60</y>
       <width>78</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Mã SV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Họ tên</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbx_tksinhvien">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>74</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Mã SV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Họ tên</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mã lớp</string>
      </property>
     </item>
    </widget>
    <widget class="QTextEdit" name="txt_tksinhvien">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>60</y>
       <width>348</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Tìm kiếm</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
